### PR TITLE
refactor(logo): add react component - INNO-1177

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Apply spec of logo.
https://webgate.ec.europa.eu/CITnet/confluence/x/kgT-MQ

Static files are now located to resources package

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 5a00b5e7c13548f59c9d6b5a9514b30a9533102a

https://deploy-preview-778--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 9d7c7af96fc9982405d552ba89b82a601ffa986e

https://deploy-preview-778--europa-component-library.netlify.com

File: src/systems/ec/implementations/react/packages/logo/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@ecl/ec-react-component-logo",
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/logo/package.json]: missing dependencies: `@ecl/ec-resources`

Comment (emeryro) [reply src/systems/ec/implementations/react/packages/logo/package.json]: fixed

**Review by yhuard — APPROVED** [review]

**Pull request merged**